Jenkinsfile

```
@@ -90,6 +91,7 @@ pipeline {
         stage('Publish tagged version') {
```

Since it can publish tagged and non-tagged versions consider this small change:
```suggestion
        stage('Publish version') {
```
It's just about definition :smile:

---

It makes sense, thanks.